.gitignore

```diff
@@ -1,5 +1,4 @@
-/crafty.js
-/crafty-min.js
+dist/
```

Comment:
Hmm good point. If we don't ignore the release artifacts, we will have to check every PR so that users don't accidentally commit their release artifacts (because of security reasons). If we do ignore the release artifacts, I believe you can still add them to the commit (by `git add dist/crafty.js`), but care has to be taken not to forget adding them on release. What do you suggest?